src/java.base/windows/native/libnio/ch/FileDispatcherImpl.c

```diff
@@ -35,7 +35,6 @@
 #include "sun_nio_ch_FileDispatcherImpl.h"
 
 #include <Mswsock.h>
-#pragma comment(lib, "Mswsock.lib")
```

Comment:
I think this came about with one of the early Microsoft contributions to have transferTo optionally use TransmitFile on Windows. This create the dependency on Mswsock.

Reply:
I believe it may have had to do with explicitly showing the dependency as David suggests, I added a comment explaining this just in case